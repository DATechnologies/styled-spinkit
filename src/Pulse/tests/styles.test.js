// @flow
import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import 'jest-styled-components'
import StyledPulse from '../styles'

test('render without props', () => {
  const wrapper = shallow(<StyledPulse size={40} color={'#333'} />)
  expect(toJson(wrapper)).toMatchSnapshot()
})
