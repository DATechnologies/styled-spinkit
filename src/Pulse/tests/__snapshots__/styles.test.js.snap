// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render without props 1`] = `
.jUCBPe {
  width: 40px;
  height: 40px;
  background-color: #333;
  margin: 40px auto;
  border-radius: 100%;
  -webkit-animation: dqpBcv 1s ease-in-out infinite;
  animation: dqpBcv 1s ease-in-out infinite;
}

<div
  className="sc-bdVaJa jUCBPe"
  color="#333"
  size={40}
/>
`;
